### PR TITLE
Update module helm charts metadata

### PR DESCRIPTION
**Pull request by froesef**
This PR sets the metadata to something similar to what the arrow-flight module has.
Also it adds the pushing of images to travis again.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -100,6 +100,13 @@ ifneq (${TRAVIS_TAG},)
 	$(call do-docker-retag-and-push-public,${TRAVIS_TAG})
 endif
 
+.PHONY: helm-push-public
+	DOCKER_HOSTNAME=$DOCKER_PUBLIC_HOSTNAME DOCKER_NAMESPACE=${DOCKER_PUBLIC_NAMESPACE} make -C modules helm-chart-push
+ifneq (${TRAVIS_TAG},)
+	DOCKER_HOSTNAME=$DOCKER_PUBLIC_HOSTNAME DOCKER_NAMESPACE=${DOCKER_PUBLIC_NAMESPACE} DOCKER_TAGNAME=${TRAVIS_TAG} make -C modules helm-chart-push
+endif
```
Comment (hunchback) [inline Makefile:107]: 1. you are missing the target `helm-push-public` but I would rather you just add the line:

```
DOCKER_HOSTNAME=$DOCKER_PUBLIC_HOSTNAME DOCKER_NAMESPACE=${DOCKER_PUBLIC_NAMESPACE} DOCKER_TAGNAME=$1 make -C modules helm-chart-push
```

to the end of macro:

```
do-docker-retag-and-push-public
```

File: modules/m4d-s3-to-s3/templates/batchtransfer.yaml [hunk 2]
```diff
@@ -4,7 +4,12 @@
 apiVersion: motion.m4d.ibm.com/v1alpha1
 kind: BatchTransfer
 metadata:
-  name: {{ .Release.Name }}
+  name: {{ include "m4d-s3-to-s3.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "m4d-s3-to-s3.name" . }}
+    helm.sh/chart: {{ include "m4d-s3-to-s3.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
```
Comment (hunchback) [inline modules/m4d-s3-to-s3/templates/batchtransfer.yaml:12]: this change is unrelated to this PR - please create a separate PR

File: Makefile (hunk 1 above)
Comment (roee88) [reply Makefile]: I added `helm-push-public`. Why do you suggest to join it with do-docker-retag-and-push-public?

**Review by roee88 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (hunchback) [reply Makefile]: its' an optimization making code more elegant ... but can be merged as-is

**Review by hunchback — APPROVED** [review]
